docs/user/quickstart.rst

```diff
@@ -42,20 +46,19 @@ started writing an API:
     # things will handle all requests to the '/things' URL path
     app.add_route('/things', things)
 
-You can run the above example using any WSGI server, such as uWSGI
-or Gunicorn. For example:
-
-.. code:: bash
+    if __name__ == '__main__':
+        with make_server('', 8000, app) as httpd:
+        print('Serving on port 8000...')
 
-    $ pip install gunicorn
-    $ gunicorn things:app
+            # Serve until process is killed
+            httpd.serve_forever()
 
-On Windows where Gunicorn and uWSGI don't work yet you can use Waitress server
+You can run the above example directly using this wsgiref server included:
```

Comment:
nitpick: It might read better if it was phrased `the wsgiref server included` or `using the included wsgiref server`